README.md

```diff
@@ -10,8 +10,6 @@ Requirements:
 - It has a .gitignore file
 - It has a README.md file to describe this project rigthly
 
-
 Thanks,
 
-
 Oscar De la Cruz
```

Comment:
- A good README is an asset that has the capability of impressing recruiters, showcasing your documentation skills and share your thoughts pattern to users/visitors.
- [ ] Your current README file is quite great but structuring it according to this [README template](https://github.com/microverseinc/readme-template) will even more impressive and professional. This is the recommended format by Microverse in order to ensure a descriptive Readme in all projects. Please ensure that you make the following changes to customize the content:

- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo.
  - The only **optional** section is FAQ.
  - It is important to add the section **Future Features**, and include 1-3 potential features of your project.
* Kindly **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and fill in your custom content.
- Another important editing is to remember to remove **template logo**. Optionally, you can replace it with your own logo if they have one.